src/MongoDB.Driver.Core/MongoDB.Driver.Core.targets

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+        <Content Include="$(MSBuildThisFileDirectory)../x64/native/windows/snappy64.dll">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            <Link>snappy64.dll</Link>
+        </Content>
+    </ItemGroup>
+</Project>
```

Comment:
Vincent mentioned in our meeting that this file is considered invasive to an application.

I assume we are keeping it only because it is the approach currently being used by the FLE C# binding, and that we are deferring finding the right way to bundle a native library in out Nuget package to a later release.